### ISSUE TITLE
Homepage/Landing Page

### ISSUE THREAD

**Issue by aneesh-neelam** [aneesh-neelam/VITacademics#99]
The homepage of the Web API requires some redesigning to make it look prettier. Currently, it looks like [this](https://vitacademics-rel.herokuapp.com/). 

As @biocross suggested, we need to put a "Share our App" function in our client apps, so that users can easily send a referral message and a link to the landing page of the API server, to their friends via messaging/email apps (OS Share function). The link is to the landing page where they can go to the respective Store page to download the app. @VinayGupta23: Please such a function under App Settings. 

@ayushagarwal95, @biocross: This is your project. The only restriction is that it should work with the [Jade Template Engine](http://jade-lang.com/).

**Comment by VinayGupta23** [comment]
Okay, I'll get it done by the next update.

**Comment by ayushagarwal95** [comment]
The design needs to be done first.

**Comment by VinayGupta23** [comment]
Okay, so should I add the link for sharing later?

**Comment by aneesh-neelam** [comment]
@VinayGupta23: No, add it now. The landing page exists already, we just want to make it prettier, especially for mobile web browsers. The URL is https://vitacademics-rel.herokuapp.com/.

**Comment by VinayGupta23** [comment]
Okay.

Sent from my Windows Phone

**Comment by aneesh-neelam** [comment]
Link the web app in the landing page.

**Comment by ayushagarwal95** [comment]
@biocross: the design for improved landing page needed. Only then can this be done.

**Comment by prijindal** [comment]
I can do it

**Comment by aneesh-neelam** [comment]
@prijindal, @ayushagarwal95: It helps if the landing page and the web app are similar in design. That'll make the whole thing look consistent.

**Comment by prijindal** [comment]
How About Something like this:
![menu](https://cloud.githubusercontent.com/assets/10034872/10353808/7207c7e8-6d77-11e5-8921-d77c9610788c.png)
![home](https://cloud.githubusercontent.com/assets/10034872/10353807/72007574-6d77-11e5-99c6-effa76e5aac8.png)

**Comment by aneesh-neelam** [comment]
@prijindal: No, looks extremely weird. Those are weird colors, use lighter ones. And don't use blue for both. Blue and blue's complement could work. 

Also, the project is called "VITacademics", note the casing, spelling and spacing.

**Comment by prijindal** [comment]
How about putting a screenshot of our apps there?

**Comment by aneesh-neelam** [comment]
@prijindal: What's the point of doing that? They are anyway present on the app pages on the various app stores. Make a page, similar to [WhatsApp :: Home](https://www.whatsapp.com/).

**Comment by prijindal** [comment]
How about this kind of layout
and Different colors
![idea_2](https://cloud.githubusercontent.com/assets/10034872/10354657/ca115f96-6d7f-11e5-9fbd-820805136195.png)

**Comment by aneesh-neelam** [comment]
Well, that's nicer. Just ensure the design is consistent throughout. 

But that's still the wrong name of the project. It's "VITacademics". And it's "Academic Metrics",  not "Academics Metric". There is more than one metric here.

**Comment by prijindal** [comment]
Great I will change that text and complete the whole prototype then

**Comment by aneesh-neelam** [comment]
@prijindal: Work with @ayushagarwal95 on this.

**Comment by prijindal** [comment]
:+1:

**Comment by rahulkapoor90** [comment]
@aneesh-neelam Was getting bored, how's this by the way?

![vit](https://cloud.githubusercontent.com/assets/11207756/10409820/29fdb062-6f4c-11e5-882e-ffcc465d0a94.png)

**Comment by rahulkapoor90** [comment]
@aneesh-neelam Was getting bored, how's this by the way?

![varwwwclientsclient1web2tmpphpf8q3sl](https://cloud.githubusercontent.com/assets/11207756/10409874/aa857ea8-6f4d-11e5-9025-6d5265c886ae.png)

**Comment by aneesh-neelam** [comment]
@rahulkapoor90: Are you insane? When I said similar to WhatsApp's website, I didn't mean an exact copy.

**Comment by rahulkapoor90** [comment]
@aneesh-neelam How can that be an exact copy it is something "similar" as you said and by similar i tried to get same layout, appreciation for work would have been nice rather than marking me as insane!

**Comment by biocross** [comment]
Aneesh, stop being an ass.

**Comment by aneesh-neelam** [comment]
I apologize for calling you insane. 

What I meant by similar was in functionality. And I was looking for inspired design. 

What you did isn't similar, it's identical in both design and functionality.

**Comment by rahulkapoor90** [comment]
@aneesh-neelam Okay, point noted.

**Comment by prijindal** [comment]
Here, I Extended My earlier design:
![idea_2_update_feb8](https://cloud.githubusercontent.com/assets/10034872/12879391/25ddaddc-ce58-11e5-85d6-1f420260a1da.png)

Sorry for the late reply.
